data/json/monsters/fish.json

```diff
@@ -1338,11 +1338,11 @@
   {
     "id": "mon_fish_blackfish",
     "type": "MONSTER",
-    "copy-from": "mon_fish_small",
+    "copy-from": "mon_fish_tiny",
     "name": { "str_sp": "black sea bass" },
     "description": "A small, robust fish, the black sea bass is known for its dark, mottled color and meaty texture.",
-    "volume": "1500 ml",
-    "weight": "600 g",
+    "volume": "1330 ml",
+    "weight": "500 g",
```

Comment:
Can grow up to 9 pounds most stay at or below 4 pounds is what I'm finding from google.  https://www.onthewater.com/black-sea-bass-fish-facts

Reply:
This one might have miscalculated weight, but not to the point of 4 lbs. Eyeballing, this could get to around 2 lbs - that I can believe, because this is a stocky fish so my calculations might be wrong.